logger/configurer/configurer.go

```diff
@@ -0,0 +1,137 @@
+package configurer
+
+import (
+	"fmt"
+	"log"
+	"time"
+
+	"github.com/coreos/go-etcd/etcd"
```

Comment:
To be the bearer of bad news... we should not be using this library anymore. CoreOS deprecated it. Instead, we should be using github.com/coreos/etcd/client.

I don't know whether it makes sense to change it on the 1.x tree. Maybe we should stay with this one for now and switch for version 2. But the new library is definitely much better.